packages/devtools_app_shared/lib/src/ui/split_pane.dart

```diff
@@ -20,9 +20,9 @@ import '../utils/utils.dart';
 ///
 /// [minSizes] defines the minimum size that each child can be set to when
 /// adjusting the sizes of the children.
-final class Split extends StatefulWidget {
+final class SplitPane extends StatefulWidget {
```

Comment:
to make this change non-breaking, can you add another widget `Split` that returns `SplitPane` in its build method. And then mark `Split` as deprecated?

Reply:
Hmm actually I see now how many places we'd have to add `hide Split` to the flutter import. Can you instead change the version number to 1.0.0? And then devtools_extensions/pubspec.yaml will need to bump its devtools_app_shared dep to 1.0.0, and also it should be bumped to 0.0.15-wip.

neither package need to be published for this to unblock the flutter PR though.

Reply:
Great we can do `0.1.0` then for `devtools_app_shared`.

To clarify what I meant:

devtools_extensions/pubspec.yaml
```yaml
version: 0.0.15-wip
...
dependencies:
  devtools_app_shared: ^0.1.0
...
```
devtools_extensions/CHANGELOG.md
```
0.0.15-wip
* Bump `devtools_app_shared` to ^0.1.0
```

Reply:
Got it, thanks. LMK if this looks good